lib/solid_telemetry/exporters/active_record/trace_exporter.rb

```diff
@@ -16,6 +16,7 @@ def export(spans, timeout: nil)
                 parent_span_id = (span_data.parent_span_id == OpenTelemetry::Trace::INVALID_SPAN_ID) ? nil : span_data.hex_parent_span_id
 
                 span_name = SpanName.find_or_create_by name: span_data.name
+                span_name.update root_span: parent_span_id.blank?
```

Comment:
Can a span name be shared between a root and non-root span?

Reply:
technically yes, a span name could be shared between a root and a non-root span. however, span names are normally meaningful. for instance, a controller action would be something like `PostsController#index` and you woudn't expect to see that span name deeper in the trace. Similarly, there are span names such as `Post.find` that you wouldn't expect to see as a root span.

That said, nothing prevents you from creating a new span and give it any name. 

In any case, we don't really care if a span name is both a root span name and deeper in the tree. This is used mostly to avoid populating the select input with non-root span names that won't return any traces.